pkg/process/v5/transformers/nvd/unique_pkg.go

```diff
@@ -63,14 +63,64 @@ func findUniquePkgs(cfgs ...nvd.Configuration) uniquePkgTracker {
 	return set
 }
 
-func determinePlatformCPEAndNodes(c nvd.Configuration) (*string, []nvd.Node) {
-	var platformCPE *string
+// nolint:gocognit
+func platformPackageCandidates(set uniquePkgTracker, c nvd.Configuration) bool {
+	nodes := c.Nodes
+	result := false
+	/*
+		Turn a configuration like this:
+		(AND (redis <= 6.1 (OR debian:8 debian:9 ubuntu:19 ubuntu:20))
+		Into a configuration like this:
+		(OR (AND redis <= 6.1 debian:8) (AND redis <= 6.1 debian:9) (AND redis <= 6.1 ubuntu:19) (AND redis <= 6.1 ubuntu:20))
+	*/
+	if len(nodes) == 2 && c.Operator != nil && *c.Operator == nvd.And {
```

Comment:
this could probably be inverted to a guard clause to help with grokking indents
```
if len(nodes) != 2 || (c.Operator != nil && *c.Operator != nvd.And) {
  return false
}
```